joomla/joomla-cms
# Add language string JAPPLY to frontend.

The [repeatable form field](https://docs.joomla.org/Repeatable_form_field_type) uses the language string "JAPPLY", which is not included in the language files for frontend.

I know that the repeatable field is not used in core and may be deprecated again soon (see #7829). But as long as we ship it, we should make sure it works, both in backend and frontend. :wink: 
#### Testing Instructions
1. Add a repeatable form field to any form .xml file that is used in frontend.
   - Example: Paste the code from [the documentation](https://docs.joomla.org/Form_field) into components/com_content/models/forms/article.xml.
2. Make sure the field is displayed by editing the according .php file.
   - Example: Add the following line at the bottom of components/com_content/views/form/tmpl/edit.php
3. View the form in frontend
   - Example: Create a menu link to submit an article in frontend and go there. You should see a "Select" button at the end of the form.
4. Click the "Select" button and note the submit button at the right bottom. Without the patch, it says "JAPPLY", with the patch "Save".
